examples/jupyter/expansion-filters-zernike.ipynb

```diff
@@ -0,0 +1,1142 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "z_n = np.array(df2['mean']) \n",
```

Comment:
Getting a column out of a dataframe will give you a `pandas.Series` object, which usually can be treated like a numpy array, so the following might work:
```suggestion
    "z_n = df2['mean']\n",
```